app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/AddPictureActivityTest.kt

```diff
@@ -59,6 +81,8 @@ class AddPictureActivityTest {
     @Before
     fun setUp() {
         Intents.init()
+        // this is a dirty trick, but will it work?
```

Comment:
This is not a helpful comment, please include the answer or reformulate.

Reply:
true, I'll reformulate